web/config/settings.py

```diff
@@ -310,6 +313,16 @@
 SLACK_CLIENT_ID = None
 
 OCTOPRINT_TUNNEL_CAP = int(os.environ.get('OCTOPRINT_TUNNEL_CAP', '1099511627776'))  # 1TB by default
+OCTOPRINT_TUNNEL_SUBDOMAIN_FMT = '{subdomain_code}.tunnels.{site.domain}'
+OCTOPRINT_TUNNEL_SUBDOMAIN_RE = re.compile(r'^(\w+)\.tunnels.*$')
+USE_OCTOPRINT_TUNNEL_V2 = (
+    os.environ.get('USE_OCTOPRINT_TUNNEL_V2', 'False') == 'True'
+)
+
+if os.environ.get('OCTOPRINT_TUNNEL_PORT_RANGE'):
```

Comment:
Let's hard-code the range to something that's unlikely to conflict, such as 53377-53477. And hardcode this port mapping in docker-compose.yaml too, so that the private server can run tunneling without changing any files.

Reply:
Let's hardcode it into the docker compose setup, then its possible to override that hardcode using env vars. Over 32k it's not very "safe", those are assigned dynamically, as far as I know. When a required port is not free, container does not start.